packages/babel-preset-expo/src/import-meta-transform-plugin.ts

```diff
@@ -0,0 +1,25 @@
+// Copyright 2015-present 650 Industries. All rights reserved.
+
+import { ConfigAPI, types } from '@babel/core';
+
+export function expoImportMetaTransformPlugin(
+  api: ConfigAPI & { types: typeof types }
+): babel.PluginObj {
+  const { types: t } = api;
+
+  return {
+    name: 'expo-import-meta-transform',
+    visitor: {
+      MetaProperty(path) {
+        const { node } = path;
+        if (node.meta.name === 'import' && node.property.name === 'meta') {
```

Comment:
Always enable this plugin and throw a babelCodeFrameError when bundling for non-server, native platforms. This will make it a bit clearer for migration purposes.

Reply:
still feel hesitant to enable this by default and we also need it for native platforms on livestore integration.
let me keep it as-is and can add a separate pr if there's anything to follow up.